### PR TITLE
feature/deseng745: Added multi-select checkboxes for project type. Cleaned up form CSS.

### PR DESCRIPTION
**Pull request by jareth-whitney**
- Added multi-select checkboxes for project type
- Cleaned up form CSS

### CODE REVIEW

File: src/app/projects/add-edit-project/add-edit-project.component.html [hunk 1]
```diff
@@ -28,87 +28,89 @@
...
-              <li *ngFor="let logo of logos.controls">
-                <div class="logo-name-container">
+            <ul class="project-logos" *ngIf="logos.controls.length > 0">
+              <li *ngFor="let logo of logos.controls" class="flex-column">
+                <div class="logo-name-container full-wide">
                   <label>File name</label>
                   <p>{{ logo.controls.name.value }}</p>
                 </div>
-                <div class="logo-link-container">
+                <div class="logo-link-container full-wide">
```
Comment (Baelx) [inline src/app/projects/add-edit-project/add-edit-project.component.html:72]: Thanks a lot for cleaning up some of the display in the backend!

File: src/app/projects/add-edit-project/add-edit-project.component.ts [hunk 2]
```diff
@@ -82,6 +82,12 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
     'Plan Implementation and Monitoring'
   ];
 
+  public projectTypes: Array<Object> = [
```
Comment (Baelx) [inline src/app/projects/add-edit-project/add-edit-project.component.ts]: You can use the ProjectTypes type that you defined in project.ts here

File: src/app/projects/add-edit-project/add-edit-project.component.ts [hunk 3]
```diff
@@ -747,14 +778,26 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
     }));
   }
 
-    /**
+  /**
+   * Takes the project type form values and retrieves the data.
+   *
+   * @returns {Array} Array of project type strings.
+   */
+  private getTypesFormValues(): Project['projectTypes'] {
```
Comment (Baelx) [inline src/app/projects/add-edit-project/add-edit-project.component.ts]: You could use the ProjectTypes type here too

**Review by Baelx — APPROVED** [review]
Thanks! Approved

File: src/app/projects/add-edit-project/add-edit-project.component.ts (hunk 2 above)
Comment (jareth-whitney) [reply src/app/projects/add-edit-project/add-edit-project.component.ts]: done

**Pull request merged**